src/pages/market/market.scss

```diff
@@ -0,0 +1,21 @@
+page-market {
+  ion-card {
+    text-align: center;
+    justify-content: 'center';
+    align-items: 'center';
+  }
+
+    .bottom{
```

Comment:
Expected indentation of 2 spaces   indentation<br>Expected single space before "{"   block-opening-brace-space-before